src/model/Util.js

```diff
@@ -0,0 +1,31 @@
+import { uuid } from "@cfworker/uuid";
+
+export class Util {
+    static FRANKLIN_UPLOAD_KV = null;
+    
+    static getUUID() {
+        return uuid();
+    }
+
+    static getURLEncoded(data) {
+        const formBody = [];
+        for (var property in data) {
+            var encodedKey = encodeURIComponent(property);
+            var encodedValue = encodeURIComponent(data[property]);
+            formBody.push(encodedKey + "=" + encodedValue);
+        }
+        return formBody.join("&");
+    }
+
+    static setCache = async (data) => await Util.FRANKLIN_UPLOAD_KV?.put('token', JSON.stringify(data))
```

Comment:
- better name
- use method syntax!

```suggestion
    static sync cacheToken(data)  {
      await Util.FRANKLIN_UPLOAD_KV?.put('token', JSON.stringify(data));
     }
```